### PR TITLE
PAYEE-387 upgrade fe  estimated taxable income journey - 011

### PR DESCRIPTION
**Pull request by masahmad**
- [ ] Unit tests passing
- [ ] Coverage reached
- [ ] Acceptance tests passing
- [ ] Reviewed

### CODE REVIEW

File: app/views/incomes/BonusPaymentAmountView.scala.html [hunk 1]
```diff
@@ -33,33 +39,57 @@
...
+
+
 @mainTemplate(
     title = messages("tai.bonusPaymentsAmount.title",TaxYearRangeUtil.currentTaxYearRangeBetweenDelimited),
-    articleClasses = None,
     authedUser = Some(user),
-    pageHeader = Some(header),
-    employerName = Some(employer.name)
+    backLinkContent = Some(Messages("tai.back-link.upper")),
+    backLinkId = "backLink"
```
Comment (sergi-hmrc) [inline app/views/incomes/BonusPaymentAmountView.scala.html:56]: why backLink?

File: app/views/incomes/BonusPaymentAmountView.scala.html [hunk 2]
```diff
@@ -33,33 +39,57 @@
         formForErrorSummary = Some(bonusOvertimeAmountForm),
         displayBackLink = true,
```
Comment (brendanstraughan) [inline app/views/incomes/BonusPaymentAmountView.scala.html:40]: should this be false?

File: app/views/incomes/BonusPaymentAmountView.scala.html (hunk 1 above)
Comment (masahmad) [reply app/views/incomes/BonusPaymentAmountView.scala.html:56]: why not? tests are probably expecting id of "backLink"  :)

File: app/views/incomes/BonusPaymentAmountView.scala.html (hunk 2 above)
Comment (masahmad) [reply app/views/incomes/BonusPaymentAmountView.scala.html:40]: yes - changed it -  btw  old backLink has no effect if showGdsVersion is set to true in header

File: app/views/incomes/BonusPaymentAmountView.scala.html [hunk 3]
```diff
@@ -33,33 +39,57 @@
...
 
+            @govukButton(Button(content = Text(messages("tai.selectIncome.link.text"))))
+
 
-            @submit()
             @cancelLink(Call("GET",controllers.routes.IncomeController.cancel(employer.id).url))
         }
     </div>
+
+        </div></div>
```
[the remark this answers is not in the document]
Comment (masahmad) [reply app/views/incomes/BonusPaymentAmountView.scala.html]: yes  done!

File: app/views/incomes/BonusPaymentAmountView.scala.html (hunk 1 above)
Comment (sergi-hmrc) [reply app/views/incomes/BonusPaymentAmountView.scala.html:56]: should we change the defaulted one to be backLink then?

Comment (masahmad) [reply app/views/incomes/BonusPaymentAmountView.scala.html:56]: i was thinking the same - there appears more "backlink" id then the current default one  - maybe another ticket for the backlog

**Review by sergi-hmrc — APPROVED** [review]

**Pull request merged**